src/main/java/seedu/address/logic/commands/csvcommand/ImportCommand.java

```diff
@@ -29,7 +30,7 @@
  * Supports bulk registration via a CSV file.
  * This command aims to facilitate registration of entities onto Alfred.
  */
-public class ImportCommand extends Command {
+public class ImportCommand extends Command implements TrackableState {
```

Comment:
In the next PR, I'm going to make Alfred create an output file with the error lines (if the user wants). So if we undo this, do we also delete the file?